telegram/_bot.py

```diff
@@ -3374,6 +3374,10 @@ async def edit_message_text(
         """
         Use this method to edit text and game messages.
 
+        Note:
+            It is currently only possible to edit messages without
+            :paramref:`reply_markup` or with inline keyboards
```

Comment:
```suggestion
            :attr:`telegram.Message.reply_markup` or with inline keyboards.
```

Same in the other places